providers/openfeature-provider-flagd/docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+services:
+  flagd:
+    build:
+      context: test-harness
+      dockerfile: flagd/Dockerfile
+    ports:
+      - 8013:8013
+  flagd-unstable:
+    build:
+      context: test-harness
+      dockerfile: flagd/Dockerfile.unstable
+    ports:
+      - 8014:8013
+  flagd-sync:
+    build:
+      context: test-harness
+      dockerfile: sync/Dockerfile
+    ports:
+      - 9090:9090
+  flagd-sync-unstable:
+    build:
+      context: test-harness
+      dockerfile: sync/Dockerfile.unstable
+    ports:
+      - 9091:9090
```

Comment:
Let's make sure this file is excluded from published package. We can do so from the `pyproject.toml`

Reply:
Good point. Checking this now it also includes `tests` and `test-harness` etc. Should we simplify to only include src?
```
[tool.hatch.build.targets.sdist]
include = [
  "src",
]
```

Reply:
I'd have to check the generated build to confirm. If you can please make this change in a separate PR.